prow/github/client.go

```diff
@@ -673,3 +674,32 @@ func (c *Client) FindIssues(query string) ([]Issue, error) {
 	}, &issSearchResult)
 	return issSearchResult.Issues, err
 }
+
+// GetFile uses github repo contents API to retrieve the content of a file with commit sha.
+// If commit is empty, it will grab content from repo's default branch, usually master.
+// TODO(krzyzacy): Support retrieve a directory
+func (c *Client) GetFile(org, repo, filepath, commit string) ([]byte, error) {
+	c.log("GetFile", org, repo, filepath, commit)
+
+	var url string
```

Comment:
This can be dried out to:
```go
url := fmt.Sprintf("%s/repos/%s/%s/contents/%s", c.base, org, repo, filepath)
if commit != "" {
   url = fmt.Sprintf("%s?ref=%s", url, commit)
}
```